pyocd/target/builtin/target_nRF5340_xxAA.py

```diff
@@ -0,0 +1,444 @@
+    0xbf004770, 0xf7ffb510, 0x4604ffe1, 0xffeaf7ff, 0xf004fb00, 0x7080f100, 0xbf00bd10, 0x42814401,
+    0x2001bf9c, 0xe0034770, 0xbf244288, 0x47702001, 0x2b04f850, 0xbf1c3201, 0x47702000, 0xbf00e7f4,
+    0xbf004770, 0x47702000, 0x47702003, 0xbf842803, 0x47702069, 0xb240b580, 0xf851a105, 0xf2400020,
+    0xf2c45104, 0x60081108, 0xf80af000, 0xbd802000, 0x00000000, 0x00000002, 0x00000001, 0x00000000,
+    0x4000f240, 0x1008f2c4, 0x29006801, 0x4770d0fc, 0x500cf240, 0x1008f2c4, 0x60012101, 0xbf00e7f0,
+    0x9000b081, 0xf04f9800, 0x600131ff, 0xe7e7b001, 0x47702069, 0xf7ffb5b0, 0x4604ffa5, 0x7f80f1b0,
+    0xf04fd90a, 0xbf007580, 0xf7ff4628, 0xf7ffffe9, 0x4405ff7d, 0xd3f742a5, 0xbdb02000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000
+    ],
+
+    # Relative function addresses
+    'pc_init': 0x21000005,
+    'pc_unInit': 0x21000039,
+    'pc_program_page': 0x210001bd,
+    'pc_erase_sector': 0x210000ad,
+    'pc_eraseAll': 0x21000089,
+
+    'static_base' : 0x21000000 + 0x00000004 + 0x0000047c,
+    'begin_stack' : 0x210024a0,
+    'begin_data' : 0x21000000 + 0x1000,
+    'page_size' : 0x800,
+    'analyzer_supported' : False,
+    'analyzer_address' : 0x00000000,
+    # Enable double buffering
+    'page_buffers' : [
+        0x210004a0,
+        0x21000ca0
+    ],
+    'min_program_length' : 0x800,
+
+    # Relative region addresses and sizes
+    'ro_start': 0x4,
+    'ro_size': 0x47c,
+    'rw_start': 0x480,
+    'rw_size': 0x4,
+    'zi_start': 0x484,
+    'zi_size': 0x10,
+
+    # Flash information
+    'flash_start': 0x1ff8000,
+    'flash_size': 0x1000,
+    'sector_sizes': (
+        (0x1000000, 0x800),
+    )
+}
+
+
+class NRF53XX(NRF53):
+    MEMORY_MAP = MemoryMap(
+        FlashRegion(
+            start=0x0,
+            length=0x200000,
+            blocksize=0x1000,
+            algo=FLASH_ALGO_APP,
+            flash_class=Flash_NRF5340,
+            core_index=0,
+        ),
+        FlashRegion(
+            start=0x01000000,
+            length=0x00040000,
+            blocksize=0x800,
+            algo=FLASH_ALGO_NET,
+            flash_class=Flash_NRF5340,
+            core_index=1,
+        ),
+        FlashRegion(
+            start=0x00ff8000,
+            length=0x1000,
+            blocksize=0x1000,
+            is_erasable=False,
+            algo=FLASH_ALGO_APP_UICR,
+            flash_class=Flash_NRF5340,
+            core_index=0,
+        ),
+        FlashRegion(
+            start=0x01ff8000,
+            length=0x800,
+            blocksize=0x800,
+            is_erasable=False,
+            algo=FLASH_ALGO_NET,
+            flash_class=Flash_NRF5340,
+            core_index=1,
+        ),
+        RamRegion(start=0x20000000, length=0x80000),
+    )
+
+    def __init__(self, session):
+        super(NRF53XX, self).__init__(session, self.MEMORY_MAP)
+        self._svd_location = SVDFile.from_builtin("nrf5340_application.svd") # TODO
```

Comment:
ideally, I would load both SVD for the specific cores. But how to do this? (seems not that important though)